lib/restforce/sobject.rb

```diff
@@ -16,6 +16,13 @@ def describe_layouts(layout_id = nil)
       @client.describe_layouts(sobject_type, layout_id)
     end
 
+    def refresh
```

Comment:
What do you think of renaming this to `#reload` to match ActiveRecord?

Reply:
I'm partial to `refresh` since it is what the Stripe & NetSuite API bindings use for this operation, but I'm happy to change it if you feel strong about the name!

Reply:
I'd like to use the ActiveRecord naming. I think it'll be more intuitive for most Ruby developers.

Reply:
Got it! Will change.

What do you think about aliasing `refresh` to the `reload` method?